Airline_Web_API/Airlina_DB/dbo/Tables/TicketTypes.sql

```diff
@@ -0,0 +1,5 @@
+﻿CREATE TABLE [dbo].[TicketTypes]
+(
+	[TicketTypeId] INT IDENTITY(1,1) NOT NULL PRIMARY KEY,
```

Comment:
As this field is inside TicketType table you don't need to duplicate this information in field name, So just [Id] is enough.